config/routes.rb

```diff
@@ -3,6 +3,7 @@
 
   root 'static_pages#index'
   resources :games, only: %i[new create show index update] do
+    put 'forfeit', on: :member
```

Comment:
Is this line supposed to be `member`? or `user`?

Reply:
I asked my mentor about it he said member was fine.